Documentation/CRDs/Block-Storage/ceph-block-pool-rados-namespace-crd.md

```diff
@@ -81,3 +81,42 @@ parameters:
   pool: replicapool
   ...
 ```
+
+### Mirroring
+
+#### Prerequisites
+
+Enable the mirroring in the default rados namespace(CephBlockPool), refer [CephBlockPool documentation](ceph-block-pool-crd.md#mirroring).
```

Comment:
Let's show a simple yaml example inline of the cephblockpool. Then they can see that it's the same pool for this rados namespace. Without the example, it's not clear that they would have to set it on the `replicapool` CephBlockPool.
```suggestion
First, enable the mirroring in the default rados namespace of the CephBlockPool that contains the new rados namespace. 
```